contrib/cai-mock-data-generator/tests/generator_test.py

```diff
@@ -0,0 +1,97 @@
+import collections
+import json
+import unittest
+import os
+
+
+from data_generator import generator
+from data_generator.file_handler import read_yaml_file
+
+
+class TestGenerator(unittest.TestCase):
```

Comment:
If we put the test here, will it get run by travis?  How about putting it with the rest of the unit tests?